app/policies/bulk_update/submissions/trainee_upload_policy.rb

```diff
@@ -4,11 +4,11 @@ module BulkUpdate
   module Submissions
     class TraineeUploadPolicy < BulkUpdate::TraineeUploadPolicy
       def show?
-        super && (trainee_upload.in_progress? || trainee_upload.succeeded?)
+        super && (trainee_upload.in_progress? || trainee_upload.succeeded? || trainee_upload.failed?)
       end
 
       def create?
-        super && trainee_upload.validated?
+        super && (trainee_upload.validated? || trainee_upload.failed?)
```

Comment:
This has been reverted in the latest changes. The thought was that if an upload `fails` due to an [`ActiveRecord::ActiveRecordError` exception](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/services/bulk_update/add_trainees/import_rows.rb#L116), they could retry to attempt resubmitting the same upload. However, since we have to investigate first what happened there would be not efficient allowing users to attempt resubmitting.